sway-ir/src/parser.rs

```diff
@@ -251,6 +252,11 @@ mod ir_builder {
                     IrAstOperation::Load(src)
                 }
 
+            rule op_log() -> IrAstOperation
+                = "log" _ log_ty:ast_ty() log_val:id() comma() _ log_id:id() {
```

Comment:
```suggestion
                = "log" _ log_ty:ast_ty() log_val:id() comma() log_id:id() {
```
The convention is whitespace is handled at the lowest rule, any call to another rule doesn't need `_` explicitly.